indico/modules/designer/client/js/index.js

```diff
@@ -159,8 +163,8 @@ import {$T} from 'indico/utils/i18n';
     }
 
     newDiv.draggable({
+      // stack: templateSide + ' > div' remove this so it does not interfere with the z-index
```

Comment:
If this is to be removed, let's just remove it. We don't usually commit commented-out code.